### PR TITLE
Add wiki-converted article on using server logs: INFODEV-2330

### PR DESCRIPTION
**Pull request by RackerKate**

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-1fefed0793/how-to/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-b91c7d9390/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-b91c7d9390/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-15d52a17a0/how-to/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-15d52a17a0/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-b2e9ac97e6/how-to/)

**Comment by RackerKate** [comment]
@cyrichardson Please review.

**Comment by cyrichardson** [comment]
On it, @RackerKate.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-1f893f726d/how-to/)

File: content/cloud-servers/using-server-logs.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+the user is coming from and then block it.
+
+### Networking logs
+
+Viewing networking logs is more complex because servers do not place this
+information in a log format unless you specifically implement the
+rules for this task.
+
+As an alternative, you can use the [Rackspace
+Intelligence](https://support.rackspace.com/how-to/rackspace-intelligence-faq/#top) monitoring tool. This tool enables you to set up a monitoring alert for
```
Comment (stephamon) [inline content/cloud-servers/using-server-logs.md]: How-to links don't need the `https://support.rackspace.com` bit. It should just be `/how-to/rackspace-intelligence-faq/#top`

**Pull request merged**